src/components/learner-credit-management/SubBudgetCard.jsx

```diff
@@ -38,8 +38,16 @@ const SubBudgetCard = ({
   const renderCardHeader = (budgetType, budgetId) => {
     const subtitle = (
       <div className="d-flex flex-wrap align-items-center">
+        <span>
+          <Badge
+            className="p-2 mr-2"
```

Comment:
[curious] Is the `p-2` class name needed? It looks like the mocks are using the normal size of the `Badge` without any additional padding.

<img width="386" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/c0ec1cbd-380c-4713-b80b-a0149eabfb77">

<img width="910" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/7e30a9ce-2df9-4d46-8088-8c3e5c2b8017">

Reply:
when I use the Badge component without padding it looks like this (a bit weird) on my local machine. What do you think now?
```js
const subtitle = (
      <Stack direction="horizontal" gap={2.5}>
        <Badge variant={budgetLabel.badgeVariant}>{budgetLabel.status}</Badge>
        <span data-testid="offer-date">
          {budgetLabel.term} {formattedDate}
        </span>
      </Stack>
    );
```
![image](https://github.com/openedx/frontend-app-admin-portal/assets/86868918/916c51f3-436c-4728-93da-3a7c1059d924)

Reply:
@jajjibhai008 Ah, that's likely because you're running the MFE locally with the default Paragon theme, not the theme that will be installed on stage/prod (i.e., `@edx/brand-edx.org`). Refer to the Paragon documentation site and switch between the "Open edX" and "edX" themes to see the difference: https://paragon-openedx.netlify.app/components/badge/

Instead of running `npm start` to run the MFE locally, I would recommend using `npm run start:with-theme` instead to ensure the `@edx/brand-edx.org` is installed for the application without risking modifying any code.

Related, while not explicitly mocked currently... the badge color variant for "Upcoming" is intended to be `secondary` as opposed to `info`, so that it more closely maps to the variants used for subscription plan status badges ([source](https://github.com/openedx/frontend-app-admin-portal/blob/9cc4dad8060c0724d6b9ae87b4fb19e2fba6e910/src/components/subscriptions/data/constants.js#L52-L56)). 

We're also shooting for parity with Subscription Management in terms of the badge label itself. That is, we should update the "Upcoming" badge to say "Scheduled" instead 😄 

Jenn from Enterprise UX will be updating Figma mocks to include budgets that are scheduled (upcoming) and expired as well.

Reply:
@adamstankiewicz Thanks for your explanation.
Here is the updated LCM page screenshot.

![screencapture-localhost-1991-test-enterprise-admin-learner-credit-2023-10-03-16_19_09](https://github.com/openedx/frontend-app-admin-portal/assets/86868918/3b660434-84c4-465e-ba87-e3234d668c30)